Makefile

```diff
@@ -162,11 +162,6 @@ out/cf-cli_linux_i686: $(GOSRC)
 	CGO_ENABLED=0 GOARCH=386 GOOS=linux go build \
 							$(REQUIRED_FOR_STATIC_BINARY) \
 							-ldflags "$(LD_FLAGS_LINUX)" -o out/cf-cli_linux_i686 .
-
-out/cf-cli_linux_x86-64: $(GOSRC)
```

Comment:
Is this intended to remove Linux for intel target?

Reply:
No, I removed the duplicate lint target which was building linux. Don't know why its showing like this

```
# lint: $(GOSRC)
# 	CGO_ENABLED=0 GOARCH=amd64 GOOS=linux go build \
# 							$(REQUIRED_FOR_STATIC_BINARY) \
# 							-ldflags "$(LD_FLAGS_LINUX)" -o out/cf-cli_linux_x86-64 .
```

Reply:
https://github.com/gururajsh/cli/commit/13aec8880f8e121e3a955b8c0dcf320425a01e89

Reply:
Looks like copy paste issue. Fixed it.

Reply:
Thanks!